release/release_gold.sh

```diff
@@ -270,36 +270,48 @@ function tag_release {
 		repository=liferay-portal
 	fi
 
-	local tag_data=$(
-		cat <<- END
-		{
-			"message": "",
-			"object": "${git_hash}",
-			"tag": "${_PRODUCT_VERSION}",
-			"type": "commit"
-		}
-		END
-	)
-
-	if [ $(invoke_github_api_post "${repository}/git/tags" "${tag_data}") -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
-	then
-		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
-	fi
+	local repository_owners=("liferay-release" "davisaints")
```

Comment:
Hey @davisaints, you should use these users only locally during your tests. :wink: 

Can you please switch to `brianchandotcom` and `liferay` instead?

Thanks